repository/rest-proxy/operator/operator.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kudo.dev/v1beta1
+name: "rest-proxy"
+version: "0.1.0"
+kudoVersion: 0.8.0
+kubernetesVersion: 1.15.0
+appVersion: 5.3.0
```

Comment:
let's update the app version to the one used in image

Reply:
Done. Thanks for the correction!